buzzmobile/process/bearing/bearing.py

```diff
@@ -46,7 +45,7 @@ def bearing(fix):
 def bearing_node():
     """Initializes bearing node."""
     rospy.init_node('bearing', anonymous=True)
-    rospy.Subscriber('/fix', NavSatFix, bearing)
+    rospy.Subscriber('fix', NavSatFix, bearing)
```

Comment:
Can you revert this to be `/fix`? No third-party nodes are namespaced, and `fix` is published by a third party node. Also see comment on subscribing to `fix` below.